conn_test.go

```diff
@@ -486,6 +471,40 @@ func TestPolicyConnPoolSSL(t *testing.T) {
 	}
 }
 
+func TestQueryTimeout(t *testing.T) {
+	srv := NewTestServer(t, protoVersion2)
+	defer srv.Stop()
+
+	cluster := NewCluster(srv.Address)
+	cluster.Timeout = 250 * time.Millisecond
```

Comment:
Is 250ms low enough - why not set it to 1ms, to make quite clear that you want the query to time out? Also, why not put a comment in the code so that we remember why we set the timeout to that particular value?